blocks/stat/stat.css

```
@@ -76,16 +90,37 @@ div.stat-container .stat::after {
 
     div.stat-container .stat > div {
         flex-direction: unset;
+        justify-content: space-between;
+        column-gap: normal;
+    }
+
+    div.stat-container .stat > div.no-effect {
+        justify-content: normal;
+        column-gap: var(--spacer-element-07);
     }
 }
 
 /* Desktop styles */
 @media (min-width: 1200px) {
     div.stat-container .stat {
-        padding: var(--spacer-layout-07) var(--spacer-layout-156) var(--spacer-layout-05);
+        padding: var(--spacer-layout-07) 0 var(--spacer-layout-05);
+    }
+
+    div.stat-container .stat > div {
+        justify-content: normal;
+        column-gap: var(--spacer-element-07);
     }
 
-    div.stat-container .stat h1 {
+    div.stat-container .stat h1,
+    div.stat-container .stat h2 {
         font-size: 128px;
```

```suggestion
        font-size: var(--font-size-128);
```

`--font-size-128: 128px;`

Let's use CSS variable for this font size so that we can easily reuse and convert to `em` unit in the future.

---

Done